workers/questionreviewer/db/repository_test.go

```
@@ -0,0 +1,211 @@
+		}
+
+		review, err := rrepo.GetReviewForQuestionOrCreateNew(context.Background(), userID, sampleQuestion.ID)
+		if err != nil {
+			t.Fatalf("unexpected error: %v", err)
+		}
+		if review.QuestionID != sampleQuestion.ID || review.UserID != userID {
+			t.Fatalf("review data mismatch: got %+v", review)
+		}
+
+		var count int
+		rrepo.DB.QueryRow(`SELECT COUNT(*) FROM question_reviews WHERE question_id=$1 AND user_id=$2`, sampleQuestion.ID, userID).Scan(&count)
```

golangci-lint (1.64.8)

Error return value of `(*database/sql.Row).Scan` is not checked

(errcheck)